src/devices/Is31fl3731/Registers.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.Display
+{
+    /// <summary>
+    /// Register addresses for the Command Register.
+    /// </summary>
+    // Enables pages one to nine
+    // Table 2 in datasheet
+    public static class CommandRegister
```

Comment:
why public cstatic lasses and not enums? you don't seem to override them or anything like this.